File: Service_eHospital_DongNai_A/Implement/Ztmp_BCTH_018_BenhAn_Noitru_LuutruService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class Ztmp_BCTH_018_BenhAn_Noitru_LuutruService : BaseService<Ztmp_BCTH_018_BenhAn_Noitru_Luutru, IZtmp_BCTH_018_BenhAn_Noitru_LuutruRepository>
    , IZtmp_BCTH_018_BenhAn_Noitru_LuutruService
    {
    private readonly IZtmp_BCTH_018_BenhAn_Noitru_LuutruRepository _Ztmp_BCTH_018_BenhAn_Noitru_LuutruRepository;
    public Ztmp_BCTH_018_BenhAn_Noitru_LuutruService(IZtmp_BCTH_018_BenhAn_Noitru_LuutruRepository Ztmp_BCTH_018_BenhAn_Noitru_LuutruRepository) : base(Ztmp_BCTH_018_BenhAn_Noitru_LuutruRepository)
    {
    _Ztmp_BCTH_018_BenhAn_Noitru_LuutruRepository = Ztmp_BCTH_018_BenhAn_Noitru_LuutruRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/Ztmp_BCTH_018_BenhAn_Noitru_Luutru.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Ztmp_BCTH_018_BenhAn_Noitru_Luutru } from './Ztmp_BCTH_018_BenhAn_Noitru_Luutru.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Ztmp_BCTH_018_BenhAn_Noitru_LuutruService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Luutru_Id', 'BenhAn_Id', 'PhongBan_Id', 'NgayVao', 'ThoiGianVao', 'NgayRa', 'ThoiGianRa', 'LyDoVao_Code', 'LyDoRa_Code'];

    List: Ztmp_BCTH_018_BenhAn_Noitru_Luutru[] | undefined;
    ListFilter: Ztmp_BCTH_018_BenhAn_Noitru_Luutru[] | undefined;
    FormData!: Ztmp_BCTH_018_BenhAn_Noitru_Luutru;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Ztmp_BCTH_018_BenhAn_Noitru_Luutru";
    }
}

